src/midi/midi_mt32.h

```diff
@@ -45,16 +45,16 @@ class MidiHandler_mt32 final : public MidiHandler {
 	static constexpr int FRAMES_PER_BUFFER = 1024; // synth granularity
 	static constexpr int SAMPLES_PER_BUFFER = FRAMES_PER_BUFFER * 2; // L & R
 
-	using render_buffer_t = std::array<float, SAMPLES_PER_BUFFER>;
-	using limited_buffer_t = std::array<int16_t, SAMPLES_PER_BUFFER>;
-	using ring_t = moodycamel::BlockingReaderWriterCircularBuffer<limited_buffer_t>;
+	using render_buffer_t = std::vector<float>;
+	using play_buffer_t = std::vector<int16_t>;
```

Comment:
I think we can stop using these two type aliases now - simple `std::vector<type>` will be easier to read and understand.

Reply:
Gone! Looking tidy :-)